native/kotlin/api/kotlin/build.gradle.kts

```diff
@@ -76,50 +76,24 @@ dependencies {
     implementation(libs.kotlinx.coroutines.core)
 }
 
+val generatedSourceDirectory = rootProject.layout.buildDirectory.get()
```

Comment:
Building up paths this way is better for cross-platform support (I can't see any reason this _shouldn't_ work on Windows, but at the moment I'm pretty sure it doesn't)